PWGHF/Core/HfMlResponseD0ToKPi.h

```diff
@@ -160,70 +177,136 @@ class HfMlResponseD0ToKPi : public HfMlResponse<TypeOutputScore>
   /// Method to get the input features vector needed for ML inference
   /// \param candidate is the D0 candidate
   /// \return inputFeatures vector
-  template <typename T1>
+  template <bool usingMl, typename T1>
```

Comment:
```suggestion
  template <bool usingMl = false, typename T1>
```
In this way the default behavior is untouched